### PR TITLE
PT-2262: Update Shipping module

### PR DESCRIPTION
**Pull request by BelyakovSergey**
## Description
Implement generic CRUD service
### Jira-link:
https://virtocommerce.atlassian.net/browse/PT-2262
### Artifact URL:

### CODE REVIEW

File: src/VirtoCommerce.ShippingModule.Data/VirtoCommerce.ShippingModule.Data.csproj [hunk 1]
```diff
@@ -20,10 +20,13 @@
...
-        <PackageReference Include="VirtoCommerce.Platform.Data" Version="3.51.0" />
+        <PackageReference Include="VirtoCommerce.Platform.Core" Version="3.62.0" />
+        <PackageReference Include="VirtoCommerce.Platform.Data" Version="3.62.0" />
     </ItemGroup>
     <ItemGroup>
         <ProjectReference Include="..\VirtoCommerce.ShippingModule.Core\VirtoCommerce.ShippingModule.Core.csproj" />
     </ItemGroup>
+    <ItemGroup>
+      <Folder Include="Caching\" />
+    </ItemGroup>
```
Comment (eMazeika) [inline src/VirtoCommerce.ShippingModule.Data/VirtoCommerce.ShippingModule.Data.csproj]: remove empty folder

File: tests/VirtoCommerce.ShippingModule.Data.Tests/VirtoCommerce.ShippingModule.Data.Tests.csproj [hunk 2]
```diff
@@ -13,7 +13,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="coverlet.collector" Version="1.3.0">
+    <PackageReference Include="coverlet.collector" Version="3.1.0">
```
Comment (eMazeika) [inline tests/VirtoCommerce.ShippingModule.Data.Tests/VirtoCommerce.ShippingModule.Data.Tests.csproj]: revert back to 1.3.0

File: tests/VirtoCommerce.ShippingModule.Data.Tests/VirtoCommerce.ShippingModule.Tests.csproj [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.1" />
+    <PackageReference Include="xunit" Version="2.4.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
+    <PackageReference Include="coverlet.collector" Version="3.1.0">
```
Comment (eMazeika) [inline tests/VirtoCommerce.ShippingModule.Data.Tests/VirtoCommerce.ShippingModule.Tests.csproj:16]: 1.3.0

Comment (BelyakovSergey) [reply tests/VirtoCommerce.ShippingModule.Data.Tests/VirtoCommerce.ShippingModule.Tests.csproj:16]: Fixed

File: src/VirtoCommerce.ShippingModule.Core/Services/IShippingMethodsSearchService.cs [hunk 4]
```diff
@@ -1,10 +1,6 @@
-using System.Threading.Tasks;
-using VirtoCommerce.ShippingModule.Core.Model.Search;
-
 namespace VirtoCommerce.ShippingModule.Core.Services
 {
     public interface IShippingMethodsSearchService
     {
-        Task<ShippingMethodsSearchResult> SearchShippingMethodsAsync(ShippingMethodsSearchCriteria criteria);
     }
```
Comment (akak1977) [inline src/VirtoCommerce.ShippingModule.Core/Services/IShippingMethodsSearchService.cs:15]: There the compatibility will be broken. Need to preserve original interface.

File: src/VirtoCommerce.ShippingModule.Core/Services/IShippingMethodsService.cs [hunk 5]
```diff
@@ -1,13 +1,6 @@
-using System.Threading.Tasks;
-using VirtoCommerce.ShippingModule.Core.Model;
-
 namespace VirtoCommerce.ShippingModule.Core.Services
 {
     public interface IShippingMethodsService
```
Comment (akak1977) [inline src/VirtoCommerce.ShippingModule.Core/Services/IShippingMethodsService.cs:11]: There the compatibility will be broken. Need to preserve original interface.

**Review by akak1977 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**